lib/code_corps/accounts/accounts.ex

```diff
@@ -56,6 +65,20 @@ defmodule CodeCorps.Accounts do
     end
   end
 
+  def upload_github_photo_async(%User{} = user) do
+    Task.Supervisor.start_child(:background_processor, fn -> upload_github_photo(user) end)
```

Comment:
I'm not sure if this needs to be explicitly async since Cloudex itself uses an async process to do the upload. Thoughts here?

Reply:
This is one of those things where I lack the experience to be sure. We should look into Cloudex internals.